dotnet/Face/sdk/identify.cs

```diff
@@ -1,83 +1,106 @@
- 
-// Define Anna
-CreatePersonResult friend1 = await faceClient.PersonGroupPerson.CreateAsync(
-    // Id of the PersonGroup that the person belonged to
-    personGroupId,    
-    // Name of the person
-    "Anna"            
-);
- 
-// Define Bill and Clare in the same way
-// </person>
-
-// <add_face>
-// Directory contains image files of Anna
-const string friend1ImageDir = @"D:\Pictures\MyFriends\Anna\";
- 
-foreach (string imagePath in Directory.GetFiles(friend1ImageDir, "*.jpg"))
+namespace ConsoleApp1
 {
-    using (Stream s = File.OpenRead(imagePath))
+    class Program
     {
-        // Detect faces in the image and add to Anna
-        await faceClient.PersonGroupPerson.AddFaceFromStreamAsync(
-            personGroupId, friend1.PersonId, s);
-    }
-}
-// Do the same for Bill and Clare
-// </add_face>
+        async static void Quickstart()
+        {
+            // <authenticate>
+            string SUBSCRIPTION_KEY = Environment.GetEnvironmentVariable("FACE_SUBSCRIPTION_KEY");
+            string ENDPOINT = Environment.GetEnvironmentVariable("FACE_ENDPOINT");
 
-// <train1>
-await faceClient.PersonGroup.TrainAsync(personGroupId);
-// </train1>
- 
-// <train2>
-TrainingStatus trainingStatus = null;
-while(true)
-{
-    trainingStatus = await faceClient.PersonGroup.GetTrainingStatusAsync(personGroupId);
- 
-    if (trainingStatus.Status != TrainingStatusType.Running)
-    {
-        break;
-    }
- 
-    await Task.Delay(1000);
-} 
-// </train2>
+            IFaceClient faceClient = new FaceClient(new ApiKeyServiceClientCredentials(SUBSCRIPTION_KEY)) { Endpoint = ENDPOINT };
+            // </authenticate>
 
-// <main>
-string testImageFile = @"D:\Pictures\test_img1.jpg";
+            // <person>
+            // Create an empty PersonGroup
+            string personGroupId = "myfriends";
+            await faceClient.PersonGroup.CreateAsync(personGroupId, "My Friends");
 
-using (Stream s = File.OpenRead(testImageFile))
-{
-    var faces = await faceClient.Face.DetectWithStreamAsync(s);
-    var faceIds = faces.Select(face => face.FaceId.Value).ToArray();
- 
-    var results = await faceClient.Face.IdentifyAsync(faceIds, personGroupId);
-    foreach (var identifyResult in results)
-    {
-        Console.WriteLine("Result of face: {0}", identifyResult.FaceId);
-        if (identifyResult.Candidates.Length == 0)
-        {
-            Console.WriteLine("No one identified");
+            // Define Anna
+            Person friend1 = await faceClient.PersonGroupPerson.CreateAsync(
+                // Id of the PersonGroup that the person belonged to
+                personGroupId,
+                // Name of the person
+                "Anna"
+            );
+
+            // Define Bill and Clare in the same way
+            // </person>
+
+            // <add_face>
+            // Directory contains image files of Anna
+            const string friend1ImageDir = @"D:\Pictures\MyFriends\Anna\";
```

Comment:
this should be some file form cloud?

Reply:
I am fixing this. Note the topic that included from this file (articles/cognitive-services/Face/Face-API-How-to-Topics/HowtoIdentifyFacesinImage.md) has since been retired (https://github.com/MicrosoftDocs/azure-docs-pr/pull/131107/commits/64cb51abb288fbef025dad31e51e9b772b31737c) so we might want to retire this file also.

Reply:
Changed to use online images.